test/unit/spec/LoginRouter_spec.js

```diff
@@ -32,11 +33,11 @@ define([
   'helpers/xhr/labels_login_ja',
   'helpers/xhr/labels_country_ja'
 ],
-function (Okta, Q, Backbone, SharedUtil, CryptoUtil, Logger, OktaAuth, Util, Expect, Router,
-          $sandbox, PrimaryAuthForm, IDPDiscoveryForm, RecoveryForm, MfaVerifyForm, EnrollCallForm, resSuccess, resRecovery,
-          resMfa, resMfaRequiredDuo, resMfaRequiredOktaVerify, resMfaChallengeDuo, resMfaChallengePush,
-          resMfaEnroll, errorInvalidToken, resUnauthenticated, resSuccessStepUp, Errors, BrowserFeatures,
-          labelsLoginJa, labelsCountryJa) {
+function (Okta, Q, Backbone, SharedUtil, CryptoUtil, Logger, DevLogger, OktaAuth, Util, Expect, Router,
```

Comment:
Can we rename the `'shared/util/Logger',` to `courageLogger` and leave this as `Logger`?